examples/vanilla/themes/minimal-theme.html

```diff
@@ -7,7 +7,7 @@
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css"
     />
-    <script src="https://cdn.jsdelivr.net/npm/@mux/mux-video@0"></script>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@mux/mux-video"></script>
```

Comment:
👍 😅 that was a blocking script, reminds me of our Mux player docs 
https://docs.mux.com/guides/mux-player-web#example-html-element-implementation
we should really put a defer on that